.gitignore

```diff
@@ -2,7 +2,6 @@ test-output
 node_modules
 helm/**/*.lock
 *.tgz
-.vscode
```

Comment:
can we had another ignore in .vscode to ignore everything but settings.json? I have a history plugin that stores history in .vscode folder

Reply:
Ah yes, good spot! I believe the best work around will be to first ignore the folder and then explicitly un-ignore that file, let me try...